README.md

```diff
@@ -86,6 +86,7 @@ dizda_cloud_backup:
     processor:
         type: tar # Required: tar|zip|7z
         options:
+            executable: zip|7z # If not added to path variable please set binary path for zip and 7z! e.g. on Windows: 'C:\Program Files\7-Zip\7z.exe'
```

Comment:
Yes the default decides which executable to use. So if `type` is set to `zip` the executable is `zip` and if its set to `7z` then the executable is `7z`. If you use `tar` as type its ignored because `tar` and `bzip` is used.